source/guides/command-line/version.md.erb

```diff
@@ -12,3 +12,13 @@ By running `hanami version` we can see the current version of the framework that
 % bundle exec hanami version
 v<%= hanami_version %>
 ```
+
+Besides there are alias `--version` and `-v`.
```

Comment:
Should be "Besides there are alias**es** `--version` and `-v`." 
But I think  "There are also aliases: `--version` and `-v`." would be even better